### DISCUSSION TITLE
Design  Changes

### DISCUSSION THREAD

**Discussion by BearGryllsTV** [tltneon/lgsl#90] [Show and tell]
Hey, 

I randomly came across this today thought I would install it and check it out, Ive made some design changes to it just to suit me.

https://my.merric-gaming.co.uk/lgsl/

Would you like me to send the changes when Im done?
![screencapture-my-merric-gaming-co-uk-lgsl-2022-01-20-11_12_44](https://user-images.githubusercontent.com/65396874/150328302-cc839305-9afd-44ce-aa2d-cb36afa0d6e9.png)

**Comment by tltneon** [comment]
Your edit looks amazing! I'd love to add it to LGSL if you don't mind.

**Comment by TacTicTow** [comment]
Nice layout. :) Can you post a zip file or something of your edits. I would like to see what you did and may implement some of it into our site.  Good job!

**Comment by BearGryllsTV** [comment]
Yeah ofc I will check through what I changed tomorrow and post it

**Reply by TacTicTow** [reply]
> Yeah ofc I will check through what I changed tomorrow and post it

Any update for this? Still working on it?
